demos/demo/src/common/Collection/computed/visibleKeys.js

```diff
@@ -1,26 +1,26 @@
-import {Computed} from 'cerebral'
+import {compute} from 'cerebral'
 import {state} from 'cerebral/tags'
 import paths from '../paths'
 import sort from '../sort'
 
 export default function (moduleName) {
   const {collectionPath, draftPath, filterPath} = paths(moduleName)
 
-  return Computed(
-    {
-      items: state`${collectionPath}.**`,
-      afilter: state`${filterPath}`,
-      selectedKey: state`${draftPath}.key`
-    },
-    ({items, afilter, selectedKey}) => {
+  return compute(
+    state`${collectionPath}`,
+    state`${filterPath}`,
+    state`${draftPath}.key`,
+    (items, afilter, selectedKey) => {
```

Comment:
Let us fix Firebare Demo in separate branch... to not block this one, other maintainers on it and I do not believe current master Firebase Demo runs correctly anyways